### PR TITLE
Port all recent commits

### PR DESCRIPTION
**Pull request by Jummit**
Updated to version 1.96.0.

### CODE REVIEW

**Comment by Merith-TK** [comment]
can you fix the conflict?

**Comment by Jummit** [comment]
> can you fix the conflict?

Sure.

**Comment by Jummit** [comment]
Done.

**Comment by Merith-TK** [comment]
I think this has some already included code, 

Im looking at the "bump version" but we are already past that??

File: gradle.properties [hunk 1]
```diff
@@ -2,7 +2,7 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Mod properties
-mod_version=1.95.3-beta
+mod_version=1.96.0-beta
```
Comment (Jummit) [inline gradle.properties:5]: Actually bumped to 1.96.0

**Comment by Jummit** [comment]
> I think this has some already included code,
> 
> Im looking at the "bump version" but we are already past that??

That's a mistake on SquidDev's part.

**Comment by Merith-TK** [comment]
please do update your repo to include the fact that `travis.yml` has been removed due to it never succeding a build

**Pull request merged**